src/filmgrain.rs

```diff
@@ -708,7 +706,7 @@ fn fgy_32x32xn_rust<BD: BitDepth>(
         };
 
         for y in ystart..bh {
-            let src = &*src_row_y(y);
+            let src: &[<BD as BitDepth>::Pixel] = &*src_row_y(y);
```

Comment:
Why do we need this?

Reply:
We don't. I think I accidentally double clicked the type annotation in vscode which inserts the inferred type... sry 😬